### PR TITLE
Aff marketing ui

### PR DESCRIPTION
**Pull request by altsop**
Merge of Affiliate Marketing UI into master branch

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -1,6 +1,4 @@
 <template>
     <require from="./css/styles.css"></require>
-    <require from="./commons/navbar-searchbar/navbar-searchbar"></require>
-    <navbar-searchbar></navbar-searchbar>
```
Comment (daysleeperx) [inline src/app.html:4]: this navbar is meant to be a common element, where the user can change apps and workspaces. Can you please revert this deletion.

**Comment by daysleeperx** [comment]
general comment: commons folder is for reusable or common components. I suggest moving your directory to src folder.

File: src/app.html (hunk 1 above)
Comment (altsop) [reply src/app.html:4]: Thanks for this comment, I have totally forgotten about that deletion. Done

**Review by niojam — APPROVED** [review]

File: src/affiliate-manager/components/admin/affiliates/allAffiliates.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import moment from 'moment';
+
+export class allAffiliates {
+
+  @bindable selectedFilter;
+  @bindable modifyUser;
+  filteredValues = [];
+  affiliatePartners;
+  allKeys;
+  httpClient = new HttpClient();
```
Comment (daysleeperx) [inline src/affiliate-manager/components/admin/affiliates/allAffiliates.js]: why not use dependency injection here?
e.g 
```javascript
@inject(HttpClient)
export class TaskService {

  constructor(httpClient) {
    this.httpClient = httpClient;
  }
...
}
```

File: src/affiliate-manager/components/admin/approve/affPartnerApprove.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import { HttpClient, json } from "aurelia-fetch-client";
+import moment from "moment";
+
+export class affPartnerApprove {
+
+  @bindable selectedFilter;
+  @bindable modifyUser;
+
+  affCandidates;
+  httpClient = new HttpClient();
```
Comment (daysleeperx) [inline src/affiliate-manager/components/admin/approve/affPartnerApprove.js]: same here

**Review by daysleeperx — CHANGES_REQUESTED** [review]

**Review by daysleeperx — APPROVED** [review]

**Pull request merged**